CMakeLists.txt

```diff
@@ -51,58 +66,87 @@ if(APPLE)
 	if(DSYMUTIL_PROGRAM)
 		foreach(lang C CXX)
 			foreach(var LINK_EXECUTABLE CREATE_SHARED_LIBRARY)
-				set(CMAKE_${lang}_${var} "${CMAKE_${lang}_${var}}" "${DSYMUTIL_PROGRAM} <TARGET>")
+				#set(CMAKE_${lang}_${var} "${CMAKE_${lang}_${var}}" "${DSYMUTIL_PROGRAM} <TARGET>")
```

Comment:
I think we still need this, because it is not the default on mac :-(

Reply:
Oops, I removed it while modifying the cmake script.
(Enabling the CXX language conditionally when using the crashpad backend, will fail cmake)
This line can be re-enabled again.